airbyte-integrations/connectors/source-google-adwords-singer/build.gradle

```diff
@@ -12,3 +12,10 @@ airbytePython {
 dependencies {
     implementation files(project(':airbyte-integrations:bases:base-singer').airbyteDocker.outputs)
 }
+
+// used to allow local iteration to work
```

Comment:
so should we be doing this in all singer sources or just this one if just this one can you explain why in the comment?

Reply:
if all of them do we need to document it? put it in the template, etc. okay if this happens in a separae commit if that's the case.

Reply:
Ideally we do all of them, but it's lower priority than getting something working. Since we only had two cases where it didn't align, and one was easily fixable, it seems easiest to do this here. If we split it for every singer source, we shouldn't do it here; it should be part of a singer integration plugin.